.github/workflows/ci.yaml

```
@@ -47,6 +47,18 @@ jobs:
               with:
                   custom-glob: packages
 
+    version_consistency:
+      name: eslint
```

I noticed a small naming issue in the CI file, how about something like this instead:

```suggestion
      name: Check version consistency of packages
```